src/app/pages/settings/settings.page.ts

```diff
@@ -19,13 +19,21 @@ export class SettingsPage implements OnInit {
   async ngOnInit() {
     await this.storage.ready();
     this.pushEnabled = await this.storage.get('pushEnabled');
+    const pushEnabledInStorage = await this.storage.get('pushEnabled');
```

Comment:
Why we do that twice ? :>

Reply:
oh crab, my error caused by rebasing, will fix it